packages/legend-query/src/stores/QueryBuilderLambdaProcessor.ts

```diff
@@ -874,18 +876,16 @@ export class QueryBuilderLambdaProcessor
   visit_LambdaFunctionInstanceValue(
     valueSpecification: LambdaFunctionInstanceValue,
   ): void {
-    valueSpecification.values
-      .map((value) =>
-        value.expressionSequence.map((expression) =>
-          expression.accept_ValueSpecificationVisitor(
-            new QueryBuilderLambdaProcessor(
-              this.queryBuilderState,
-              this.precedingExpression,
-            ),
+    valueSpecification.values.map((value) =>
+      value.expressionSequence.map((expression) =>
+        expression.accept_ValueSpecificationVisitor(
+          new QueryBuilderLambdaProcessor(
+            this.queryBuilderState,
+            this.precedingExpression,
           ),
         ),
-      )
-      .flat();
+      ),
+    );
```

Comment:
Hm, since don't even return anything, I get it that we can remove `flat()`, but then why do we need to do `.map()` and not `.forEach()`?